content/blog/wingingit/winging-it-10.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Learn CSS Grid First
+episode: 10
+sponsors: false
+tags:
+  - CSS
+  - Grids
+  - Layout
+author:
+  - miriam
+  - stacy
+date: 2024-07-23
+length: 54 mins
+image:
+  src: winging-it/winging-it-10.jpg
+media:
+  - youtube: aDMWD_CYpEI
+summary: |
+  We talk with Stephanie Eckles
+  – the brain behind ModernCSS.dev and SmolCSS.css
+  – about CSS grids.
```

Comment:
```suggestion
  -- the brain behind ModernCSS.dev and SmolCSS.css
  -- about CSS grids.
```

Since it's hard to tell the difference in monospace fonts, we generally let 'typogr' create dashes for us.